source/guides/stacks/dlrs.rst

```diff
@@ -44,7 +46,7 @@ Releases
 
 Refer to the `Deep Learning Reference Stack website`_ for information and download links for the different versions and offerings of the stack.
```

Comment:
Ln 40: 
Correct usage is:  
Intel® Xeon® Scalable processor; or 
Intel® Xeon® Scalable processor family

It depends on what you intend. I believ you mean the first.

Reply:
I did mean the first. TY!